code/modules/events/disease_outbreak.dm

```diff
@@ -83,11 +83,13 @@
 	announce_when = ADV_ANNOUNCE_DELAY
 	///The disease type we will be spawning
 	var/datum/disease/virus_type
+	///The preset (classic) or generated (advanced) illness name
+	var/datum/disease/illness_type
```

Comment:
why are you casting `illness_type` if all you do is read the name string? just do `var/illness_type = ""`

Reply:
Fixed.